lib/ruby2d/text.rb

```diff
@@ -11,6 +11,10 @@ def initialize(x=0, y=0, text="Hello World!", size=20, font=nil, c='white', z=0)
       
       # if File.exists? font
         @font = font
+      end
+      
+      # if File.exists? font
+        # @font = font
       # else
```

Comment:
Sorry, I'm confused. What should we replace lines 12-18 with here? Just `@font = font`? (Currently, we have a trailing `end` on line 14 that's breaking the build.)